seed/data_importer/match.py

```diff
@@ -595,19 +595,25 @@ def states_to_views(unmatched_state_ids, org, access_level_instance, cycle, Stat
         if "ubid" in matching_criteria:
             check_jaccard = bool(matching_criteria.get("ubid"))
             ubid = matching_criteria.pop("ubid")
+            region = ubid.split("+")[0]
```

Comment:
checking region is not a 100% guarantee. Might want to put this particular change on hold